File: 0001 Code Chef/0014 Add two numbers/implementation/0009 dart/add_two_number.dart
import 'dart:io';

void main(){
  String? string_input = stdin.readLineSync();
  if(string_input != null){
    int? test = int.tryParse(string_input);
      while(test != null && test > 0){
        String? user_input = stdin.readLineSync();
        if(user_input != null){
          List<String> string_array = user_input.split(" ");
          int? x = int.tryParse(string_array[0]);
          int? y = int.tryParse(string_array[1]);
          if(x != null && y != null){
            print(x + y);
          }
        }
        test = test - 1;
    }
  }
}  

File: 0001 Code Chef/0014 Add two numbers/implementation/0007 ruby/AddTwoNumber.rb
stringInput = gets.chomp
test = stringInput.to_i
while(test > 0)
    stringInput = gets.chomp
    stringArray = stringInput.split(" ");
    x = stringArray[1].to_i
    y = stringArray[0].to_i
    puts (x+y)
    test = test - 1
end